src/main/riddl/Order/Order.riddl

```diff
@@ -35,7 +34,7 @@ context OrderContext is {
   command CancelOrder {orderId: OrderId, cancellingMember: MemberId}
   event OrderCancelled {orderId: OrderId, info: OrderInfo, meta: OrderMetaInfo}
   query GetOrderInfo {orderId: OrderId, requstingMembert: MemberId}
-  result OrderInfoResult {orderId: OrderId, info: OrderMetaInfo}
+  result OrderInfoResult {orderId: OrderId, info: OrderInfo}
 
```

Comment:
To get a complete picture of the order we may want both OrderInfo and OrderMetaInfo
result OrderInfoResult {orderId: OrderId, info: OrderInfo, meta: OrderMetaInfo}